pkg/cmd/utilities.go

```diff
@@ -56,7 +56,7 @@ func exportResults(outputProvided bool, fileName string, contents string, logger
 }
 
 func writeToFile(fileName string, contents string, logger zap.SugaredLogger) error {
-	logger.Debug("Writing to file...")
+	logger.Infof("Vulnerability snapshot saved at %s", fileName)
```

Comment:
That's a great idea. Users may want to see a confirmation that their command ran successfully 👍

Reply:
Yeah, I used the `snyk-to-html` cli to get an idea of what content it produces and saw it output the text when I saved the content to a file so I thought it would be good to mirror what it was doing.  The fact you had already added a log statement was a bonus :)